doc/locale/ja/LC_MESSAGES/reference/commands/status.po

```diff
@@ -232,3 +232,20 @@ msgstr "このGroongaプロセスのバージョンです。"
 
 msgid "``5.0.7``"
 msgstr ""
+
+msgid "``features``"
+msgstr ""
+
+msgid ""
+"The list of features that Groonga has and whether they are enabled or not."
+msgstr "Groongaの機能と、その機能が有効かどうかの一覧です。"
+
+msgid "``apache_arrow``"
+msgstr ""
+
+msgid ""
+"The information about Apache Arrow that Groonga uses. This item is displayed "
```

Comment:
```suggestion
"The version information of Apache Arrow that Groonga currently uses. Only displayed when Apache Arrow is enabled. "
```
To make specifically indicate the information is about "Version". Also it is clear enough this Apache Arrow is enable in Groonga so extract that part.

Reply:
@HashidaTKS You need to update `.rst` too.

Reply:
Sorry for my delay, I have updated .rst.